ios/IoReactNativeSecureStorage.swift

```diff
@@ -1,8 +1,142 @@
 @objc(IoReactNativeSecureStorage)
-class IoReactNativeSecureStorage: NSObject {
 
-  @objc(multiply:withB:withResolver:withRejecter:)
-  func multiply(a: Float, b: Float, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
-    resolve(a*b)
+/// React Native bridge which leverages ``SecureStorage`` functionality to store content securly via Keychain.
+/// Methods are run in a custom serial queue to syncronize access to the Keychain, thus only one block can run at any time to
+/// syncronize the Keychain access.
+/// Each string passed to the `put` and `get` is encoded in UTF-8.
+class IoReactNativeSecureStorage: NSObject {
+  
+  private typealias ME = ModuleException
+  
+  private let storage = SecureStorage(serviceName: "rn-secure-storage");
+  
+  let queue = DispatchQueue(label: "rn-secure-storage-queue")
+  
+  /// See ``SecureStorage/put(key:data:)``
+  /// `data` is encoded in UTF-8.
```

Comment:
What do you think to add also params?

```suggestion
  /// `data` is encoded in UTF-8.
  ///   - resolve: The promise resolve block to call with the result.
  ///   - reject: The promise reject block to call in case of an error.
```

Reply:
I would say that's not necessary as you are not directly providing the resolve/reject method, if that makes sense to you.